docs/api/spec/components/schemas/configconnectionprofile.yaml

```diff
@@ -141,7 +141,7 @@ ConfigConnectionProfile:
       type: string
     network_logoff:
       default: ~
-      description: This allows users to access the network logoff page (http://packetfence.packetfence.org/networklogoff)
+      description: This allows users to access the network logoff page (http://172-105-101-170.ip.linodeusercontent.com/networklogoff)
```

Comment:
We should avoid publishing this in the spec. Can we use PCRE post-generation to replace public IPv4 with a placeholder?